src/sentry/issues/json_schemas.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Mapping
+
+EVENT_PAYLOAD_SCHEMA: Mapping[str, Any] = {
+    "type": "object",
+    "properties": {
+        # required properties
+        "event_id": {"type": "string", "minLength": 1},
+        "platform": {"type": "string", "minLength": 1},
+        "project_id": {"type": "integer"},
+        "tags": {"type": "object"},
+        "timestamp": {"type": "string", "format": "date-time"},
+        "message_timestamp": {"type": "string", "format": "date-time"},
+        # "title": {"type": "string", "minLength": 1}, leaving this out, for now
+        # non-required properties
+        "breadcrumbs": {"type": "object"},
+        "contexts": {"type": "object"},
+        "dist": {
+            "type": "object",
+            "properties": {
+                "dist": {"type": "string", "minLength": 1},
+                "release": {"type": "string", "minLength": 1},
+            },
+            "required": ["dist", "release"],
+        },
```

Comment:
this is a bit different from https://develop.sentry.dev/sdk/event-payloads/types/#request but it's in our internal spec

Reply:
is this different from https://github.com/getsentry/sentry-data-schemas/blob/main/relay/event.schema.json ?

Reply:
We're going to be handling things manually at first because we're monkeying a bit with a specific payload. Down the line will get ours to be in line with the main event payload (and probably add a couple attributes)

Reply:
got it! thanks! :)

Reply:
Wasn't aware of that schema, we probably want to use it in the future